### ISSUE TITLE
insurance app bugs

### ISSUE THREAD

**Issue by sandeepHiscope** [sandeepHiscope/VDrPreProduction#32]
1. accessibility is missing, add buttons to navigate back and forth
2. remove dark mode compatibility 
3. 

![Image](https://github.com/user-attachments/assets/fcaa97f9-84d6-4024-b0ec-01fe9e61c883)

![Image](https://github.com/user-attachments/assets/7b89e967-0cd0-4726-a9b9-f57c87741cf6)

![Image](https://github.com/user-attachments/assets/f3a2f98a-41ae-440d-98ac-cd55de6616c2)

4. make all backgrounds white to get good experience
5. give proper alignment in form2 ![Image](https://github.com/user-attachments/assets/b05bc9fb-2d7f-45d5-8da8-d1418872ffe9)\
6. make this image border less to seamlessly integrate into background ![Image](https://github.com/user-attachments/assets/7341f017-d41a-418f-8927-2c8326b38efb)
7. 

![Image](https://github.com/user-attachments/assets/214b772c-366e-4c68-a342-67a3b2cd65da)
